### ISSUE TITLE
Face swap error with DreamShaperXL_Turbo, useing upscale or var + image prompt as face target just after around 30 sec get ^C ( useing colab gradio live T4 free user)

### ISSUE THREAD

**Issue by Amit30swgoh** [lllyasviel/Fooocus#1619]
![image](https://github.com/lllyasviel/Fooocus/assets/42430868/011c93e6-e10f-46bf-8589-119e1e5a3582)

Ps i tried both methods and got same error
https://youtu.be/nWvOzBG1k_k?si=sZJU-5KG8L3zdWMp

[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 3
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 1.4
[Parameters] Seed = 9147341485204343132
[Fooocus] Downloading control models ...
[Fooocus] Loading control models ...
extra clip vision: ['vision_model.embeddings.position_ids']
[Parameters] Sampler = dpmpp_sde - karras
[Parameters] Steps = 6 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.transformer.text_model.embeddings.position_ids'}
left over keys: dict_keys(['conditioner.embedders.0.logit_scale', 'conditioner.embedders.0.text_projection'])
Base model loaded: /content/Fooocus/models/checkpoints/dreamshaperXL_turboDpmppSDE.safetensors
Request to load LoRAs [['None', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [/content/Fooocus/models/checkpoints/dreamshaperXL_turboDpmppSDE.safetensors].
Requested to load SDXLClipModel
Loading 1 new model
^C

**Comment by mashb1t** [comment]
Most likely your RAM is exhausted, which is why your process is cancelled. Please check and upgrade it or your plan.

See https://github.com/lllyasviel/Fooocus/issues/1330, https://github.com/lllyasviel/Fooocus/issues/1574, https://github.com/lllyasviel/Fooocus/issues/1558#issuecomment-1868166226

**Comment by Amit30swgoh** [comment]
> Most likely your RAM is exhausted, which is why your process is cancelled. Please check and upgrade it or your plan.
> 
> See #1330, #1574, [#1558 (comment)](https://github.com/lllyasviel/Fooocus/issues/1558#issuecomment-1868166226)

ty friend
